src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+
+    def publish_manifest(self):
+        manifest = self.manifest.dict()
+        if not manifest["name"]:
```

Comment:
If name was not in the manifest, I believe `self.manifest` would already raise a `ValidationError`, no?
We should leave this method commented out, but at the same time, we _could_ probably remove the `if not` checks here since they happen at the model-level now.

Reply:
Actually, `name` and `version` field are optional in spec, but recommended/required for publishing
https://github.com/ApeWorX/ethpm-types/blob/770ba756947215a58a41391c71cbe717708e8ab8/ethpm_types/manifest.py#L58-L60

Reply:
Is this resolved since this code has now been commented out for the time being?